.rubocop.yml

```diff
@@ -32,3 +32,7 @@ Rails/UnknownEnv:
 
 Naming/VariableNumber:
   EnforcedStyle: snake_case
+
+Rails/Output:
+  Exclude:
+    - "app/services/importers/*"
```

Comment:
Why do we need to add it? So that `puts` is not flagged by rubocop?

In `SchoolDataImporter` we use `Rails.logger` instead - probably worth keeping logging mechanism consistent.

Reply:
I run this on the console so kind of want to see it in stdout and not lost somewhere in the logs

Reply:
```
/app # /usr/local/bin/bundle exec rails c
Loading production environment (Rails 6.1.3.2)
irb(main):001:0> Rails.logger.info "Test"
I, [2021-08-02T09:14:41.794245 #727]  INFO -- : Test
=> 53
```
Not sure what `53` is in there, but... This comes from our production app. `Rails.logger` still outputs to stdout, I think.

Reply:
If I did change to `Rails.logger` I don't see how logging the output would be very beneficial though?

Reply:
It would remove the rubocop unhappiness 😂 

And I guess it would push us more in the direction of consistency of logging in our services / tasks.

I don't really mind the functionality.